disnake/threads.py

```diff
@@ -111,9 +111,21 @@ class Thread(Messageable, Hashable):
         Bots and users with :attr:`~Permissions.manage_channels` or
         :attr:`~Permissions.manage_messages` bypass slowmode.
     message_count: Optional[:class:`int`]
-        An approximate number of messages in this thread. This caps at 50.
+        An approximate number of messages in this thread.
+
+        .. note::
+
+            If the thread was created before July 1, 2022, this stops counting at 50.
```

Comment:
take a look at discord/discord-api-docs#5437 for the wording here

Reply:
> take a look at discord/discord-api-docs#5437 for the wording here

ok I'll fix it